projects/swimlane/ngx-charts/src/lib/common/base-chart.component.ts

```diff
@@ -209,4 +210,48 @@ export class BaseChartComponent implements OnChanges, AfterViewInit, OnDestroy,
 
     return results;
   }
+
+  toDataURL<T extends 'png' | 'jpg' | 'svg'>(
+    options: {
+      type?: T;
+      canvasOptions?: { pixelRatio?: number; transparentBackground?: boolean };
+    } = {}
+  ): Promise<string> {
+    if (this.getContainerDims() === null)
+      // If not browser
+      return null;
```

Comment:
Include brackets to match coding style.
```suggestion
    if (this.getContainerDims() === null) {
      // If not browser
      return null;
    }
```